### PR TITLE
IDE-4124 don't appear watch popup that if module don't support watch …

### PR DESCRIPTION
**Pull request by haoyisun**
IDE-4124 don't appear watch popup that if module don't support watch task

### CODE REVIEW

File: tools/plugins/com.liferay.ide.gradle.ui/src/com/liferay/ide/gradle/ui/WatchableProjectPropertyTester.java [hunk 1]
```diff
@@ -35,6 +38,19 @@ public boolean test(Object receiver, String property, Object[] args, Object expe
 			IFile buildFile = project.getFile("build.gradle");
 
 			if (LiferayWorkspaceUtil.inLiferayWorkspace(project) || LiferayWorkspaceUtil.isValidWorkspace(project)) {
+
```
Comment (jtydhr88) [inline tools/plugins/com.liferay.ide.gradle.ui/src/com/liferay/ide/gradle/ui/WatchableProjectPropertyTester.java]: sf

File: tools/plugins/com.liferay.ide.gradle.ui/src/com/liferay/ide/gradle/ui/WatchableProjectPropertyTester.java [hunk 2]
```diff
@@ -35,6 +38,19 @@ public boolean test(Object receiver, String property, Object[] args, Object expe
...
+				IPath projectLocation = project.getLocation();
+
+				File projectFile = projectLocation.toFile();
```
Comment (jtydhr88) [inline tools/plugins/com.liferay.ide.gradle.ui/src/com/liferay/ide/gradle/ui/WatchableProjectPropertyTester.java]: use FileUtil.getFile(path)

**Comment by jtydhr88** [comment]
see comment and resend, thanks.

**Comment by haoyisun** [comment]
got it

**Pull request closed without merging**